tests/unit/components/Login-test.js

```diff
@@ -1,6 +1,7 @@
 "use strict"
 
 import { mount } from '@test'
+import sinon from 'sinon'
```

Comment:
How come you need to import sinon here in Ligin-test.js, but you did not have to import it in ArtefactMenuItem-test.js?  I assume sinon.spy worked in ArtefactMenuItem-test.js since you added a whopping 11% to our code coverage in one fell swoop!

Reply:
Ah! Good catch. My guess is that is actually taken care of by one of the karma plugins. We probably shouldn't import `sinon`. This can just be removed in that case at some point.

Reply:
Thanks for the clarification.